addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -4,6 +4,7 @@
 from odoo.exceptions import UserError
 from odoo.fields import Command
 from odoo.tools import format_date, frozendict
+from odoo.tools.misc import formatLang
```

Comment:
For 17.0:
```suggestion
from odoo.tools import format_date, formatLang, frozendict
```
Starting from 17.4, you can remove the `format_date` import, as it's unused there.